src/backend/functions/projects.ts

```diff
@@ -201,4 +201,4 @@ const handler: Handler = async (event, context) => {
   }
 };
 
-export { handler };
+export { handler };
```

Comment:
I notice that you have several files where the new line at the end has been removed, contrary to the prettier formatting rules. You might want to check your prettier and vscode settings to see about turning on format on save for prettier to help out fix this. There are instructions in the [Onboarding docs](https://github.com/Northeastern-Electric-Racing/PM-Dashboard-v2/blob/main/docs/Onboarding.md) for how to set up prettier.